remo/featuredrep/views.py

```diff
@@ -10,11 +11,25 @@
 from models import FeaturedRep
 
 
-@cache_control(private=True, max_age=60*10)
+@cache_control(private=True, max_age=60 * 10)
 def list_featured(request):
     """List all Featured Reps."""
+    featured_list = FeaturedRep.objects.all().order_by('-created_on')
```

Comment:
The order of the entries in the queryset is already sorted be the `updated_on` field here [1], so there is no need for this.

[1] https://github.com/mozilla/remo/blob/master/remo/featuredrep/models.py#L25